styles.css

```diff
@@ -91,24 +91,25 @@ main {
   color: #FFF;
   background-repeat: no-repeat;
   background-size: 25px 25px;
-  background-position: 60px 2px;
-  height: 50px;
-  width: 150px;
+  background-position: 58px 10px;
```

Comment:
Nice job with the attention to detail! Thank you for making sure it matches the comp!